client/src/components/App.js

```diff
@@ -8,6 +8,8 @@ import RegistrationForm from "./registration/RegistrationForm.js";
 import SignInForm from "./authentication/SignInForm.js";
 import TopBar from "./layout/TopBar.js";
 import FrontPage from "./FrontPage.js"
+import PuzzleShow from "./puzzleShow.js"
+import PuzzlePage from "./puzzlePage.js"
```

Comment:
the names of `puzzlePage` and `puzzleShow` should be capitalized! This would stick out to me pretty quickly in a review.